platform/darwin/test/MGLStyleLayerTests.h

```diff
@@ -6,5 +6,15 @@
 
 @property (nonatomic) IBOutlet MGLMapView *mapView;
 @property (nonatomic) XCTestExpectation *expectation;
+@property (nonatomic, copy, readonly, class) NSString *layerType;
```

Comment:
I figured we only run the tests on Xcode 8, so there's no harm in using it here. If we decide to drop support for Xcode 7.3, that needs to be a more deliberate decision, and it's probably too close to the release to make that change. We just recently dropped support for Xcode 7.2 in v3.3.7.